src/helpers/floating.ts

```diff
@@ -37,3 +39,25 @@ export const getArrowPlacement = ({ placement }: { placement: Placement }): Plac
     left: 'right',
   }[placement.split('-')[0]] as Placement;
 };
+
+export const mergeWrapperClassName = ({
```

Comment:
You created this one because it is used in two places, right? I'm wondering if this case isn't worth repeating the code. But not against leaving it as a helper. I'm just thinking loud.

Reply:
@rluders I actually didnt like this utility function either. I will repeat the twMerge in both places then